distrobox-init

```diff
@@ -458,6 +458,16 @@ if ! grep -q "${container_user_name}" /etc/passwd; then
 		--gid "${container_user_gid}" \
 		"${container_user_name}"; then
 
+		printf "Warning: there was a problem setting up the user\n"
+	fi
+else
+	if ! usermod \
```

Comment:
Given that we're adding usermod command, it would be nice if you add it to the list of checks in lines 194-197